storage/blockchain/src/service/write.rs

```diff
@@ -69,8 +76,7 @@ fn write_block(env: &ConcreteEnv, block: &VerifiedBlockInformation) -> ResponseR
         Err(e) => {
             // INVARIANT: ensure database atomicity by aborting
             // the transaction on `add_block()` failures.
-            TxRw::abort(tx_rw)
-                .expect("could not maintain database atomicity by aborting write transaction");
+            TxRw::abort(tx_rw).expect(TX_RW_ABORT_FAIL);
```

Comment:
These invariant comments (all of them) should be removed since the `const` documents the panic reasoning now.